src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_PatternSwitchStatement.cs

```diff
@@ -9,295 +9,445 @@
 using System.Collections.Generic;
 using System;
 using Roslyn.Utilities;
+using Microsoft.CodeAnalysis.CSharp;
 
```

Comment:
I'll clean up the usings in this source file.

---
In reply to: [70324393](https://github.com/dotnet/roslyn/pull/12416#discussion_r70324393) [](ancestors = 70324393)